packages/qwik-city/runtime/src/link-component.tsx

```diff
@@ -48,7 +48,10 @@ export const Link = component$<LinkProps>((props) => {
           prefetchSymbols(url.pathname);
 
           if (elm.hasAttribute('data-prefetch')) {
-            loadClientData(url, elm, { prefetchSymbols: false });
+            loadClientData(url, elm, {
+              prefetchSymbols: false,
```

Comment:
I'm a little bit confused about `prefetchSymbols` being `false`!

Reply:
Isn't this specifically to fetch the data separately? The symbols are being fetched elsewhere.

Reply:
> Isn't this specifically to fetch the data separately?

I don't know really. Currently the prefetch only data. and symbols are prefetched via the `service-worker` maybe.

Reply:
Actually it's done two lines above this call